VideoLocker/src/main/java/org/edx/mobile/view/my_videos/MyRecentVideosFragment.java

```diff
@@ -477,13 +480,15 @@ private void onConfirmDelete() {
         if (list != null) {
             for (SectionItemInterface section : list) {
                 if (section.isDownload()) {
+                    // TODO The removeDownload() triggers a callback upon video deletion.
+                    // Would be better if removeDownload() could take a list of videos to delete.
                     DownloadEntry de = (DownloadEntry) section;
                     environment.getStorage().removeDownload(de);
                     deletedVideoCount++;
+                    adapter.remove(section);
```

Comment:
Hmm, this removes the video items, but does not remove the section header when it becomes empty:
![shot](https://cloud.githubusercontent.com/assets/1685613/15935545/9fd3d0fc-2e35-11e6-824e-4d9dbdcb28a3.png)